tests/GraphQl/Type/SchemaBuilderTest.php

```diff
@@ -122,7 +122,7 @@ public function testGetSchema(bool $paginationEnabled)
         $schema = $mockedSchemaBuilder->getSchema();
         $queryFields = $schema->getConfig()->getQuery()->getFields();
 
-        $this->assertEquals([
+        $this->assertSame([
```

Comment:
Why not `assertEquals`?

Reply:
`assertSame` is the best practice (`assertEquals` is like `==` while `assertSame is like `===`). +1 to use `assertSame`.